### PR TITLE
adding project

### PR DESCRIPTION
**Pull request by JeduardoOJ**
Adding project

### CODE REVIEW

File: .github/workflows/workspace.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Workspace
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```
Comment (newpavlov) [inline .github/workflows/workspace.yml]: Change to `main`.

File: workflows/workspace.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: Workspace
+
+on:
```
Comment (newpavlov) [inline workflows/workspace.yml]: Remove this file

**Review by newpavlov — CHANGES_REQUESTED** [review]

File: .github/workflows/workspace.yml (hunk 1 above)
Comment (JeduardoOJ) [reply .github/workflows/workspace.yml]: done!

File: workflows/workspace.yml (hunk 2 above)
Comment (JeduardoOJ) [reply workflows/workspace.yml]: Done

**Review by newpavlov — APPROVED** [review]

**Pull request merged**